.github/workflows/pyci.yml

```diff
@@ -0,0 +1,25 @@
+name: tests
```

Comment:
wdyt about renaming this file to `tests.yml`? imo, `pyci` is too vague and doesn't describe what's going on here